src/main/java/com/raoulvdberge/refinedstorage/apiimpl/network/NetworkNodeGraph.java

```diff
@@ -30,11 +31,7 @@ public NetworkNodeGraph(TileController controller) {
     }
 
     @Override
-    public void rebuild(BlockPos start, boolean notify) {
-        if (start == null) {
-            start = controller.getPosition();
-        }
-
+    public void rebuild() {
```

Comment:
I think you forgot about the interdimensional network system in your refactor, as I don't think it works anymore.